docs/enterprise-setup/multi-region.md

```diff
@@ -0,0 +1,308 @@
+| `enabled`      | Set this to true.                                                                                                   |
+
+For additional request examples, see [the API reference](https://reference.airbyte.com/reference/regions#/).
+
+</details>
+
+<details>
+  <summary>Response</summary>
+
+Make note of your `dataplaneId`, `clientId` and `clientSecret`. You need them to associate your workspace to a region.
+
+```json title="200 Successful operation"
+json
+{
+  "dataplaneId": "uuid-string",
+  "clientId": "client-id-string",
+  "clientSecret": "client-secret-string"
+}
+```
+
+</details>
+
+## 3. Associate a region to a workspace {#step-3}
+
+One you have a region and a data plane, you need to associate that region to your workspace. You can associate a workspace with a region when you create that workspace or later, after it exists.
+
+:::note
+You can only associate each workspace with one region.
+:::
+
+### When creating a new workspace
+
+<details>
+  <summary>Request</summary>
+
+Send a POST request to /v1/workspaces/
+
+```bash
+curl -X POST "https://example.com/api/public/v1/workspaces" \
+  -H "Authorization: Bearer YOUR_ACCESS_TOKEN" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "name": "My New Workspace",
+    "dataResidency": "auto"
+  }'
+```
+
+Include the following parameters in your request.
+
+| Body parameter  | Description                            |
+| --------------- | -------------------------------------- |
+| `name`          | The name of your workspace in Airbyte. |
+| `dataResidency` | A string with a region identifier.      |
+
+</details>
+
+<details>
+  <summary>Response</summary>
+
+```json
+{
+  "workspaceId": "uuid-string",
+  "name": "workspace-name",
+  "dataResidency": "auto",
+  "notifications": {
+    "failure": {},
+    "success": {}
+  }
+}
+```
+
+</details>
+
+### When updating a workspace
+
+<details>
+  <summary>Request</summary>
+
+Send a PATCH request to /v1/workspaces/`{workspaceId}`
+
+```bash
+curl -X PATCH "https://example.com/api/public/v1/workspaces/{workspaceId}" \
+  -H "Authorization: Bearer YOUR_ACCESS_TOKEN" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "name": "Updated Workspace Name",
+    "dataResidency": "us-west"
+  }'
+```
+
+Include the following parameters in your request.
+
+| Body parameter  | Description                            |
+| --------------- | -------------------------------------- |
+| `name`          | The name of your workspace in Airbyte. |
+| `dataResidency` | A string with a region identifier.      |
+
+</details>
+
+<details>
+  <summary>Response</summary>
+
+```json
+{
+  "workspaceId": "uuid-string",
+  "name": "updated-workspace-name",
+  "dataResidency": "region-identifier",
+  "notifications": {
+    "failure": {},
+    "success": {}
+  }
+}
+```
+
+</details>
+
+## 4. Update your deployment values {#step-4}
```

Comment:
I think you should be creating a new values file since this is a separate Kubernetes Deployment. It's also missing a few fields I think , such as the enterprise license key, the control plane URL